_docs/services/s3.md

```diff
@@ -100,18 +100,25 @@ For example, you might want to use the [AWS Command Line Interface](https://aws.
 ```sh
 #!/bin/bash
 
+Run this script with a . in order to set environment variables in your shell
```

Comment:
For readability, it might be better to use `source` instead of `.`, but OK to ship as far as I'm concerned.

Reply:
I was going to do that, but I discovered that only the "." syntax is portable across shells.

Reply:
https://stackoverflow.com/a/20097303/17138235

Reply:
https://ss64.com/bash/source.html#:~:text=source%20is%20a%20synonym%20for,available%20after%20the%20script%20completes.